checker/tests/nullness/java8inference/Issue402.java

```diff
@@ -3,13 +3,14 @@
 
 // @below-java8-jdk-skip-test
 
-// This test now seems to fail because of Issue 1006
-// @skip-test
-
 import java.util.Comparator;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
+@SuppressWarnings({
+    "nullness",
+    "keyfor"
+}) // Type argument inference does not infer the correct types.
```

Comment:
Ok, but the current location satisfies neither having the explanation with the `@SuppressWarnings` nor with the key.
You had stated 'Our standard is to put the @SuppressWarnings explanation on the same line with the @SuppressWarnings annotation'.
Anyways, you seem to insist on this location so let's keep it.